### PR TITLE
Replace dart2aot with dart2native.

### PR DESCRIPTION
**Pull request by godofredoc**
This is because dart2aot does not exist in the new dart sdk.

### CODE REVIEW

File: packages/fuchsia_ctl/tool/build.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ mkdir -p build
 
 tool/dart-sdk/bin/pub get
 
-tool/dart-sdk/bin/dart2aot bin/main.dart build/main.aot
+tool/dart-sdk/bin/dart2native -k aot bin/main.dart -o build/main.aot
 rm -f build/main.aot.dill
 cp -f tool/dart-sdk/bin/dartaotruntime build/
```
Comment (dnfield) [inline packages/fuchsia_ctl/tool/build.sh]: Do we still need `dartaotruntime`?

Comment (jonahwilliams) [reply packages/fuchsia_ctl/tool/build.sh]: No, dart2native should produce a single binary

Comment (dnfield) [reply packages/fuchsia_ctl/tool/build.sh]: In that case I think we want to name it differently.  We can probably just ditch the wrapper script as well.

Comment (godofredoc) [reply packages/fuchsia_ctl/tool/build.sh]: Done!  updated the script to create a native binary and deleted the wrapper script.

Task execution using led:

https://chromium-swarm.appspot.com/task?id=48beb14ee4497310

**Pull request merged**